app/Model/AppModel.php

```diff
@@ -106,14 +106,15 @@ public function afterSave($created, $options = array())
     // this could become useful in the future
     public function updateMISP($command)
     {
+        $db_update_success = false;
         switch ($command) {
             case '2.4.20':
-                $this->updateDatabase($command);
+                $db_update_success = $this->updateDatabase($command);
```

Comment:
I think variable names should be `camelCase` in MISP.

Reply:
Hey @JakubOnderka thanks for the feedback (I just saw it..)!
Indeed, ``camelCase`` is widely used in the MISP codebase.
I must admit that I used the ``under_score`` notation because of personal taste but also because the rule is not applied everywhere (there are mixes between ``cameCase`` and ``under_score`` in a lot of places).

Even though (AFAIR) PSR-x does not explicitly specify the notation to be used on variable, I agree that having an homogeneous  code base would be better.

What do you think?